src/styles/styles.js

```diff
@@ -361,6 +361,18 @@ const styles = {
         backgroundColor: themeColors.offline,
     },
 
+    floatingActionButton: {
+        backgroundColor: colors.green,
```

Comment:
Let's use a themeColor for this instead of the absolute color value - so `themeColors.buttonSuccessBG`

Reply:
Done!